### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by armansito**
Bare bones homepage for the paper. Contains links to arXiv, this GitHub repository, embedded YouTube video of the 40 second reel, and the abstract.

Some things to add in the short term:
- A bit more detail about the code and how to find the shader code in Vello.
- A natbib citation blurb
- Embedded version of https://levien.com/tmp/beztoy/

### CODE REVIEW

**Comment by raphlinus** [comment]
Definitely looks good as a start. I think the HTML file (and any other assets it links to, including the wasm for the interactive example) should be in a `docs/` subdirectory, otherwise it mixes up serving source. I'm fine with this starting out as static HTML, and *possibly* going to an action to deploy later if we need it, but there's also a pretty good chance we won't.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <title>GPU-friendly Stroke Expansion</title>
+  <meta charset="UTF-8">
+  <meta property="og:description" content="GPU-friendly Stroke Expansion"/>
+  <link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
+  <style>
+    body {
+      font-family: 'Lato', Verdana, Helvetica, sans-serif;
```
Comment (raphlinus) [inline index.html]: The GF call to bring in Lato is missing, so this renders as Verdana on my system.

Comment (armansito) [reply index.html]: Done.

**Comment by armansito** [comment]
I moved files relating to the landing page to `landing_page/`. I can rename that to `docs/` too.

I also embedded beztoy with minor modifications. The description for the toy could use a bit more info (e.g. it doesn't explain the red error bar atm). I built the WASM locally (using https://github.com/raphlinus/raphlinus.github.io/pull/100) and simply included the build output here.

**Review by raphlinus — APPROVED** [review]
Looks great! I agree it might be nice to have more description (like the error bar), but I think we can start here.

There's probably a way to use a folder name other than `/docs`, but I can't immediately figure it out. The subdirectories that are there now (`flatten` etc) don't show in the config screen. So I recommend we change the subdirectory to that.

I also like the Lato font well enough :)

**Pull request merged**